include/mrdox/Platform.hpp

```diff
@@ -11,6 +11,8 @@
 #ifndef MRDOX_PLATFORM_HPP
 #define MRDOX_PLATFORM_HPP
 
+#include <llvm/Config/llvm-config.h>
```

Comment:
I rather not do this here, it is my hope and dream that the public API which is needed for `Generator` does not require LLVM.